src/common/log.ts

```diff
@@ -6,6 +6,10 @@ import log from 'electron-log';
 
 import Util from 'common/utils/util';
 
+// Turn off sync logging to prevent blocking the main thread
+// One downside to this is that some logs may not be written to the log file when the app closes
```

Comment:
To expand on this, any logs that need to be written after the `handleAppBeforeQuit` listener fires seem to be missed. Most of those are debug logs though, so it won't affect many cases, but it's important to be aware in case we need to debug some issue with the app on quit.

Reply:
Could you re-enable sync logs when before-quit is fired?

Reply:
Tried that - unfortunately didn't help. At the time the Node process is quitting any logs are already in the queue to write to file, and they get lost when the process ends. You'd think they'd have a built-in way to wait for those to finish :P

Reply:
Yeah you'd think there'd be some sort of `.flush()`. Too bad!